main.js

```diff
@@ -210,6 +274,79 @@ function animate(renderer, labelRenderer, camera, controls) {
     labelRenderer.render(scene, camera);
 }
 
+function clearGameBoard(scene, gameBoard) {
```

Comment:
Nicely scoped function. Looks like its a very appropriate abstraction around a well defined task, with a clear name. These are all nice indicators of the "right" abstraction, if such a thing exists!